packages/tradershub/src/components/CFDSection/CFDHeading/CFDHeading.tsx

```diff
@@ -1,7 +1,8 @@
 import React, { Fragment } from 'react';
 import { useHistory } from 'react-router-dom';
 import { useIsEuRegion } from '@deriv/api';
-import { Button, Heading, qtMerge, Text, useBreakpoint } from '@deriv/quill-design';
+import { Button, Heading, qtMerge, useBreakpoint } from '@deriv/quill-design';
```

Comment:
`Heading` is still being used in this file can we remove it and use `Text` instead from `deriv-com/ui`

Reply:
@aizad-deriv I wanted to update the Heading by Text, but I have some troubles :) H4 size is 18px, but Text component has just these sizes:

<img width="313" alt="image" src="https://github.com/binary-com/deriv-app/assets/120570511/9d44379c-48de-4181-8119-05cf1b281bbb">

So, I'll leave it as Heading for now